### ISSUE TITLE
Questions about the best model evaluation results

### ISSUE THREAD

**Issue by deyang2000** [tum-traffic-dataset/coopdet3d#9]
Hi! I am trying to use CoopDet3D. I attempted to utilize the model file coopdet3d_vi_cl_pointpillars512_2xtestgrid_yolos_transfer_learning_best.pth for testing on both the validation and training sets of the CoopDet3D VI dataset. However, the results I am getting look quite different from the figures shown in the table. I'm not sure if I am doing something wrong, but I'm a bit confused about this discrepancy. Could it be related to labeling differences?

val:
![Image](https://github.com/user-attachments/assets/1ac45a61-c813-47fd-98ea-4e56d7b4166b)
train:
![Image](https://github.com/user-attachments/assets/47690b3d-e518-4055-83e8-3c91c236f2ed)

**Comment by Shanxing1** [comment]
Hi, I also found the same problem, have you solved this problem?
BTW, have you tried to visualise the labels by using the dev-kit? I found it's not aligned with images when I tried to visualise it.

**Comment by chunyeow** [comment]
Same here. Any solution?

**Comment by Qqianliu** [comment]
same here， dou You find some solutions？

**Comment by chunyeow** [comment]
@Qqianliu @Shanxing1 @deyang2000 Anythings that you all have found out that worth to further explore. But seems remain the same here

**Comment by wanghangege** [comment]
The best performance I got with single 3090 was lower, maybe affected by batch etc., but it was too different from the original results in the paper. This confused me.
